parlai/agents/seq2seq/seq2seq.py

```diff
@@ -538,7 +547,7 @@ def batch_act(self, observations):
         if is_training:
             report_freq = 0
         else:
-            report_freq = 0.01
+            report_freq = 0.001
```

Comment:
it seems like this should be a param, you keep changing it?

Reply:
I mean it's just a logging param, so changing it via the cmdline is kind of annoying anyways

Reply:
you changing it every PR is annoying...!! lol

Reply:
you would have a default on the param, so it wouldnt change anything